packages/schema/docs/build/schema.md

```diff
@@ -1141,6 +1145,35 @@ A unique identifier for this item.
 
 -----
 
+## /LockObjectSchema
```

Comment:
Gave the lock object its own schema to make the description of its fields visible in this file. Also, it's easier to provide examples and anti-examples here than nesting it in the BasicOperationSchema.